spec/unit/type/schedule_spec.rb

```diff
@@ -542,6 +552,10 @@ def min(method, count)
     end
   end
 
+  it "should raise an error if the weekday is an int higher than 6" do
+    expect { @schedule[:weekday] = 7 }.to raise_error(Puppet::Error)
```

Comment:
Can you add the message text, e.g. `raise_error(Puppet::Error, /some message string/)`. Otherwise puppet might raise a `Puppet::Error` unrelated to the one we care about.

Reply:
There are several instances of this in the file elsewhere so I was using this based on those. Isn't it dangerous to match based on strings? What if the string ends up changing?

Reply:
Yeah there is some duplication, but if the code changes we at least know to update the string, which I think is an improvement over a (possible) false positive.